web/static/help/Messaging_es.md

```diff
@@ -0,0 +1 @@
+../../../doc/help/Messaging_es.md
```

Comment:
oh and the `../../../doc/help/Messaging_es.md` is because is a symlink the same as Messaging.md